google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -3547,4 +3547,36 @@ HmacKeyMetadata updateHmacKeyState(
    * @throws StorageException upon failure
    */
   ServiceAccount getServiceAccount(String projectId);
+
+  /**
+   * Creates a notification with the specified entity on the specified bucket.
+   *
+   * @return the notification that was created.
+   * @throws StorageException upon failure
+   */
+  Notification createNotification(String bucket, NotificationInfo notificationInfo);
+
+  /**
+   * Get the notification with the specified name on the specified object.
+   *
+   * @return the notification object that exist on the bucket.
+   * @throws StorageException upon failure
+   */
+  Notification getNotification(String bucket, String notification);
+
```

Comment:
Very confusing doc. I guess you mean bucket, not object. You mention 'the specified name' and parameter 'notification' without any documentation. 
It's not clear what is 'notification name', may be you mean 'topic'?

`the notification object that exist on the bucket.`  What happens if the object does not exist? 
A good spec should cover that.